src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -374,6 +374,7 @@ private boolean getEvent(HttpServletRequest request, HttpServletResponse respons
     }
     response.setContentType("application/json;");
     
+    event.setOwnerID(userStorageObject.getUsernameByID(event.getOwnerID()));
```

Comment:
It seems to me that this is a bad practice, since we use the ownerId with other scopes in some places. But since you are not updating the storage, should be fine